README.md

```diff
@@ -1,84 +1,104 @@
 # preCICE-adapter for the CFD code ANSYS Fluent
-Developed by Bernhard Gatzhammer Update-Try to newer Fluent and Precice Versions
-
-1. How to build the Fluent-preCICE adapter: 
-* Put the library of preCICE (libprecice.so) into the lnamd64 folder.
-* Adapt lnamd64/2ddp_host/user.udf line 1 "CSOURCES=...": There are several main udf files
-    - fsi_udf.c: For FSI simulations. Needs fsi.c.
-    - wave_profile_udf.c: For wave simulations with inflow profile. Needs 
-      wave_profile.c.
-    - wave_maker_udf.c: For wave simulations with moving wall. Needs 
-      wave_maker.c.
-    - fsi_and_wave_profile_udf.c: For FSI + wave simulations. Needs fsi.c and 
-      wave_profile.c.
-    The variable SOURCES needs one one main udf file and the correspoding .c files.
-    
-* Adapt the path of the python library in /src/makefile line 19
-* Type: make "FLUENT_ARCH=lnamd64" to start the build. Add a "clean" to clean it.
+*Developed by Bernhard Gatzhammer and valid for FLUENT 19.1 and preCICE v1.4*
+
+## 1. How to build the Fluent-preCICE adapter: 
+  * Put the library of preCICE (libprecice.so) into the lnamd64 folder.
+  * Adapt lnamd64/2ddp_host/user.udf line 1 "CSOURCES=...": There are several main udf files
+      The variable SOURCES needs one main udf file and the correspoding .c files.
+      - fsi_udf.c: For FSI simulations. Needs fsi.c.
+      - wave_profile_udf.c: For wave simulations with inflow profile. Needs 
+        wave_profile.c.
+      - wave_maker_udf.c: For wave simulations with moving wall. Needs 
+        wave_maker.c.
+      - fsi_and_wave_profile_udf.c: For FSI + wave simulations. Needs fsi.c and 
+        wave_profile.c.
+
+  * Adapt lnamd64/2ddp_host/user.udf line 3 "FLUENT_INC = ..." to the fluent installation
+    folder which will be of the type *./ansys_inc/v191/fluent*  
+  * Adapt the path of the python library in /src/makefile line 19
+  * Update the Ansys RELEASE version in /src/makefile line 26
+  * Type: make "FLUENT_ARCH=lnamd64" to start the build. Add a "clean" to clean it.
```

Comment:
Put code in ticks for better formatting: `make "FLUENT_ARCH=lnamd64"`

For cleaning, write the full command. Is it something like `make clean "FLUENT_ARCH=lnamd64"` or does `make clean` suffice?

Reply:
Further below, there are some more locations, where ticks should be used for code instructions. For paths we usually also use `code formatting` (e.g. in the wiki), but I am not sure whether there is a convention on this.